### PR TITLE
feat(plasma-web/b2c): Merge Grid component

### PR DESCRIPTION
**Pull request by Yakutoc**
Плановое объеденение компонентов. 

Упростили и теперь из `plasma-hope` идет **re-export** компонента **Grid** в 

- b2c
- web

:baby_chick: Download canary assets:

[colors--canary.436.4517103454.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.436.4517103454.xml)  
[color_metro_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.436.4517103454.swift)  
[color_metro_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.436.4517103454.kt)  
[color_metro_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.436.4517103454.ts)  
[color_sberHealth_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.436.4517103454.swift)  
[color_sberHealth_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.436.4517103454.kt)  
[color_sberHealth_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.436.4517103454.ts)  
[color_sbermarket_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.436.4517103454.swift)  
[color_sbermarket_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.436.4517103454.kt)  
[color_sbermarket_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.436.4517103454.ts)  
[color_sberprime_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.436.4517103454.swift)  
[color_sberprime_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.436.4517103454.kt)  
[color_sberprime_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.436.4517103454.ts)  
[color_selgros_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.436.4517103454.swift)  
[color_selgros_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.436.4517103454.kt)  
[color_selgros_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.436.4517103454.ts)  
[color_smbusiness_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.436.4517103454.swift)  
[color_smbusiness_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.436.4517103454.kt)  
[color_smbusiness_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.436.4517103454.ts)  
[PlasmaTokensColor--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.436.4517103454.swift)  
[shadow_sbermarket_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.436.4517103454.ts)  
[typo_mage_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.436.4517103454.swift)  
[typo_mage_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.436.4517103454.kt)  
[typo_mage_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.436.4517103454.ts)  
[typo_plasma_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.436.4517103454.swift)  
[typo_plasma_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.436.4517103454.kt)  
[typo_plasma_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.436.4517103454.ts)  
[typo_ruler_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.436.4517103454.swift)  
[typo_ruler_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.436.4517103454.kt)  
[typo_ruler_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.436.4517103454.ts)  
[typo_sage_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.436.4517103454.swift)  
[typo_sage_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.436.4517103454.kt)  
[typo_sage_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.436.4517103454.ts)  
[typo_sbermarket_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.436.4517103454.swift)  
[typo_sbermarket_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.436.4517103454.kt)  
[typo_sbermarket_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.436.4517103454.ts)  
[typo_soulmate_ios-swift--canary.436.4517103454.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.436.4517103454.swift)  
[typo_soulmate_kotlin--canary.436.4517103454.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.436.4517103454.kt)  
[typo_soulmate_react-native--canary.436.4517103454.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.436.4517103454.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.167.0-canary.436.4517103454.0
  npm install @salutejs/plasma-hope@0.25.0-canary.436.4517103454.0
  npm install @salutejs/plasma-web@1.192.0-canary.436.4517103454.0
  # or 
  yarn add @salutejs/plasma-b2c@1.167.0-canary.436.4517103454.0
  yarn add @salutejs/plasma-hope@0.25.0-canary.436.4517103454.0
  yarn add @salutejs/plasma-web@1.192.0-canary.436.4517103454.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-436/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-436/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-436/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-436/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-436/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-436/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-436/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-436/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-436/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-436/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-436/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-436/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-436/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-436/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-436/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-436/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-436/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-436/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-436/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-436/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-436/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-436/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-436/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-436/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-436/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-436/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-436/temple-storybook/

File: packages/plasma-b2c/src/components/Grid/Container.tsx [hunk 1]
```diff
@@ -1,10 +1,4 @@
 import styled from 'styled-components';
-import { Container as BaseContainer } from '@salutejs/plasma-hope';
+import { Container as ContainerHope } from '@salutejs/plasma-hope';
 
-/**
- * Блок с полями по бокам для размещения контента по вертикали.
```
Comment (Yeti-or) [inline packages/plasma-b2c/src/components/Grid/Container.tsx:5]: потерялось https://plasma.sberdevices.ru/web/components/grid/#container ?

File: packages/plasma-hope/src/components/Grid/views/b2c/Container.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+import { Container as BaseContainer } from '../web/Container';
+
+export const Container = styled(BaseContainer)`
+    max-width: 125rem;
```
Comment (Yeti-or) [inline packages/plasma-hope/src/components/Grid/views/b2c/Container.tsx:6]: Screenshot 2023-03-23 at 19 41 10
кстати а мы можем же объединить это совсем, я в душе не помню зачем нам разные сетки на  web/b2c, только давай отдельную задачу про это

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-436/

File: packages/plasma-hope/src/components/Grid/views/b2c/Container.tsx (hunk 2 above)
Comment (Yakutoc) [reply packages/plasma-hope/src/components/Grid/views/b2c/Container.tsx:6]: @Yeti-or Завел задачу - PLASMA-1937

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-436/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-436/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-436/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-436/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-436/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-436/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: packages/plasma-hope/src/components/Grid/Container.tsx [hunk 3]
```diff
@@ -1,35 +1,20 @@
...
+    b2c: ContainerB2C,
+};
 
 /**
  * Блок с полями по бокам для размещения контента по вертикали.
  * Блок нельзя вкладывать сам в себя или дальше по дереву.
  */
-export const Container = styled.div`
-    margin: 0 auto;
+export const Container: FC<PropsWithChildren<{ design: 'b2c' | 'web' }>> = (props): JSX.Element => {
```
Comment (Yeti-or) [inline packages/plasma-hope/src/components/Grid/Container.tsx:15]: все ещё отсюда не подхватывается комментарий 
https://plasma.sberdevices.ru/pr/pr-436/web/components/grid/

Comment (Yakutoc) [reply packages/plasma-hope/src/components/Grid/Container.tsx:15]: @Yeti-or Да, ты прав. Уже исправил.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-436/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-436/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-436/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-436/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-436/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-436/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-436/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-436/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-436/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-436/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-436/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-436/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-436/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-436/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Pull request merged**